actioncable/lib/action_cable/remote_connections.rb

```diff
@@ -45,6 +45,7 @@ def disconnect
         end
 
         # Returns all the identifiers that were applied to this connection.
+        remove_method :identifiers
```

Comment:
Thank you. It does seem strange, and I'm sure we can, but I'm getting lost in the code. You don't just mean renaming it, do you?